queries.sql

```diff
@@ -8,3 +8,83 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+
+/*The BEGIN statement starts a new transaction. The UPDATE statement updates all rows in the 'animals' table
+by setting the 'species' column to 'unspecified'. The SELECT statement verifies that the change was made
+by selecting all rows where the 'species' column is 'unspecified'. The ROLLBACK statement rolls back the transaction,
+undoing the changes made by the UPDATE statement. The final SELECT statement verifies that the 'species' column
+went back to its original state by selecting all rows where the 'species' column is not 'unspecified'.*/
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals; -- verify the change
+ROLLBACK;
```

Comment:
* Nice job but according to the [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=roll%20back%20the%20change%20and%20verify%20that%20the%20species%20columns%20went%20back%20to%20the%20state%20before%20the%20transaction.) after rolling back the change verify that the species columns went back to the state before the transaction.